packages/components/psammead-brand/src/__snapshots__/index.test.jsx.snap

```diff
@@ -78,6 +78,10 @@ exports[`Brand should render correctly with link not provided 1`] = `
   }
 }
 
+@media (max-width:14.9375rem) {
+
+}
+
```

Comment:
I'm seeing lots of snapshot additions like this. Please can we confirm why they're appearing?